BootloaderCorePkg/Stage2/Stage2.c

```diff
@@ -98,6 +96,15 @@ PreparePayload (
     }
   }
 
+  if (ContainerSig < COMP_TYPE_INVALID) {
+    SignatureBuf = SIGNATURE_32 (' ', ' ', ' ', ' ');
+  } else {
+    SignatureBuf = ContainerSig;
+  }
+  DEBUG ((DEBUG_INFO, "Loading Payload ID %4a", (CHAR8 *)&SignatureBuf));
+  SignatureBuf = ComponentName;
+  DEBUG ((DEBUG_INFO, ":%4a\n", (CHAR8 *)&SignatureBuf));

```

Comment:
Thanks adding this print. 
Better adding this kind of pint to containLib so that it could cover all the container components including IPFW components?

Reply:
ContainLib is already using this print format to print string instead of HEX.
For example:
DEBUG ((DEBUG_INFO, "Registering container %4a\n", (CHAR8 *)&SignatureBuffer));

Reply:
I don't remember it prints the container name and component name when loading TCC components.
Better also add this info to LoadComponent()?

Reply:
It does print like this: From ADL logs captured at different places:
Registering container KEYH
Registering container IPFW
Registering container EPLD

Reply:
thx!   Anyway, let me add more print to print both container name and component name.

Reply:
Thanks for the update.